core/isolate.rs

```diff
@@ -68,15 +68,21 @@ pub enum StartupData<'a> {
   None,
 }
 
-/// Defines the how Deno.core.dispatch() acts.
-pub trait Dispatch {
+#[derive(Default)]
+pub struct Config {
+  dispatch: Option<Arc<Fn(&[u8], deno_buf) -> (bool, Box<Op>) + Send + Sync>>,
```

Comment:
Note for later: I don't think dispatch() should return an Option.
We could opt (muhaha) to panic if the dispatcher doesn't recognize an op, or alternatively it could dispatch to "UnknownOp" which would always respond with an error.

Reply:
I think there are many cases where you would want to use the deno crate but not want to implement ops.. For example if you wanted to make a very simple d8 style program. It's also nice to have small examples of using the isolate without having to also implement dispatch.

Reply:
If the dispatcher isn't set in `Config` currently `Deno.core.dispatch()` causes a panic. But I think it would be cool, in the future, to have `Deno.core.dispatch()` throw a JS error instead.